### PR TITLE
fix: duplicate comments for dependency pull requests

### PR DESCRIPTION
**Pull request by PengTian0**
The comments are duplicated for dependency pull requests:
image

### CODE REVIEW

File: build-release-tools/application/post-result.py [hunk 1]
```diff
@@ -109,7 +107,7 @@ def generate_failure_report(build_url, build_name):
     
     return None
 
-def get_sub_builds(build_url, depth = 1):
+def get_sub_builds(build_url, FAIL_REPORTS, depth = 1):
```
Comment (yyscamper) [inline build-release-tools/application/post-result.py]: is there any reason you have to use the capital letters for the argument `FAIL_REPORTS`? If no, please change to lower case just like others.

Comment (PengTian0) [reply build-release-tools/application/post-result.py]: Thanks!
Fixed.

**Comment by PengTian0** [comment]
@anhou @yyscamper @panpan0000 @changev

**Review by anhou — APPROVED** [review]

**Comment by panpan0000** [comment]
👍 
in the long run, we could avoid using global variable as passenger between functions

**Pull request merged**